### ISSUE TITLE
Feature Request - Shiny toggle

### ISSUE THREAD

**Issue by frannievas** [FlagBrew/PKSM#983]
**TL; DR** 
Pokemon shiny toggle

**Is your feature request related to a problem? Please describe.**
I was thinking in some kind of toggle option for making the pokemon shiny or not. 

**Describe the solution you'd like**
This will need to modify the PID (*pokemon ID*). One way of do it can be:
> https://www.smogon.com/ingame/rng/pid_iv_creation#how_shiny

Split the PID in two 16-bit numbers. That is, if the PID was [465DB901], it would split it in two as [465D] and [B901]. We shall call the first 4 hexadecimal digits the HID (high ID) and the last 4 hexadecimal digits the LID (Low ID). Now convert the HID, LID, Trainer ID (which we shall denote by TID for short) and Secret ID (denoted by SID for short) to binary, and compare the first 13 bits of all four, like so:
```
HID: xxxxxxxxxxxxx|xxx
LID: xxxxxxxxxxxxx|xxx
TID: xxxxxxxxxxxxx|xxx
SID: xxxxxxxxxxxxx|xxx
```
[...]
We now count the number of bits (*by columns*) in the first bit, second bit, third bit ... up to the thirteenth bit of each of these four 16-bit numbers. 
For example:
```
 HID: 1011010110001|111
 LID: 0000101100101|010
 TID: 1010100100011|000
 SID: 0001011110111|011
Ones: 2022222420224
```
Since counting the ones of each bit up to the thirteenth one in turn never yields a number that is not 1 or 3, the Pokemon will be shiny!

**Describe alternatives you've considered**
Some alternatives for people that don't want to mess with IV's or other stuffs related to the PID's
Change TID (*trainer ID*) or SID (*Secret id*)

**Additional context**
I can make the algorithm, what i don't know is how everything it's put together in your code.
If you don't like the feature that i'm proposing, it would be awesome if you can point me in the right direction for compiling and building the project on my own so i can fork the project and make the changes there.

P/D: sorry for my English skills

**Comment by piepie62** [comment]
Already exists

**Comment by frannievas** [comment]
Can you explain please?

**Comment by piepie62** [comment]
https://camo.githubusercontent.com/72a6d31e51e0832a71899a6141bb37977a87ee4b/68747470733a2f2f692e696d6775722e636f6d2f3147517353626c2e706e67 Look at this screenshot

**Comment by frannievas** [comment]
I'm sorry. I'm using the spanish interface and the translation it's not accurate. That's why i dont know about it.
Thank you any way

**Comment by piepie62** [comment]
Ah, that's fair. Could you fix that translation, then?

**Comment by frannievas** [comment]
Of course, that will be awesome. There is some "correct way" to make a PR with the changes? Do you have some standard to follow?

**Comment by piepie62** [comment]
All you should need to do is edit this line to put the correct string in https://github.com/FlagBrew/PKSM/blob/master/assets/romfs/i18n/es/gui.json#L394

**Comment by piepie62** [comment]
While you're at it, I would really appreciate it if you could add/translate any untranslated/missing strings from https://github.com/FlagBrew/PKSM/blob/master/assets/romfs/i18n/en/gui.json

**Comment by frannievas** [comment]
@piepie62 I just made a PR #984 with missing strings. If everythings it's ok i would keep adding missings strings and checking existings ones.
